ui.apps/src/main/content/jcr_root/apps/core/cif/components/commerce/productlist/v2/productlist/_cq_dialog/.content.xml

```diff
@@ -53,6 +58,46 @@
                         uncheckedValue="false"
                         value="true"
                         checked="{String}true"/>
+                    <fragmentEnabled
```

Comment:
What do we need this toggle for? Wouldn't it be enough to consider "fragments enabled" if at least one is configured?